keys/employees-keys/README.md

```diff
@@ -116,10 +133,159 @@ sub   4096R/CC8BE1D5 2021-04-02
+
+
+### Key signing
+
+3mdeb Team Leaders and Management should carefully confirm identity, if any
+concern arise it should be reported to higher management.
+
+Check received key fingerprint:
+
+```shell
+gpg -n -q --import --import-options import-show your-name-key.asc
+```
+
+If it match fingerprint received by other channel and identity was confirmed
+you can proceed with signing. First import key:
+
+```shell
+gpg --import your-name-key.asc
+```
+
+Check signatures before signing:
+
+```shell
+$ gpg --check-sigs your.name@3mdeb.com
+gpg: checking the trustdb
+gpg: marginals needed: 3  completes needed: 1  trust model: pgp
+gpg: depth: 0  valid:   7  signed:   6  trust: 0-, 0q, 0n, 0m, 0f, 7u
+gpg: depth: 1  valid:   6  signed:   1  trust: 6-, 0q, 0n, 0m, 0f, 0u
+gpg: next trustdb check due at 2021-11-23
+pub   rsa4096 2021-04-07 [SC] [expires: 2023-04-07]
+      D9DEEABFF447B80F7EC03A4BBAA0A4837C891E29
+uid           [ultimate] Your Name <your.name@3mdeb.com>
+sig!3        BAA0A4837C891E29 2021-04-07  Your Name <your.name@3mdeb.com>
+sub   rsa4096 2021-04-07 [E] [expires: 2023-04-07]
+sig!         BAA0A4837C891E29 2021-04-07  Your Name <your.name@3mdeb.com>
+
+gpg: 2 good signatures
+```
+
+Signing should look as follows. If fingerprint match then confirm with `y`.
+
+```shell
+$ gpg -u piotr.krol@3mdeb.com --sign-key your.name@3mdeb.com
+
+sec  rsa4096/BAA0A4837C891E29
+     created: 2021-04-07  expires: 2023-04-07  usage: SC  
+     trust: ultimate      validity: ultimate
+ssb  rsa4096/4F81AE572F9EFECA
+     created: 2021-04-07  expires: 2023-04-07  usage: E   
+[ultimate] (1). Your Name <your.name@3mdeb.com>
+
+
+sec  rsa4096/BAA0A4837C891E29
+     created: 2021-04-07  expires: 2023-04-07  usage: SC  
+     trust: ultimate      validity: ultimate
+ Primary key fingerprint: D9DE EABF F447 B80F 7EC0  3A4B BAA0 A483 7C89 1E29
+
+     Your Name <your.name@3mdeb.com>
+
+This key is due to expire on 2023-04-07.
+Are you sure that you want to sign this key with your
+key "Piotr Król <piotr.krol@3mdeb.com>" (B2EE71E967AA9E4C)
+
+Really sign? (y/N) y
+```
+
+Check signatures after signing:
+
+```shell
+$ gpg --check-sigs your.name@3mdeb.com
+gpg: checking the trustdb
+gpg: marginals needed: 3  completes needed: 1  trust model: pgp
+gpg: depth: 0  valid:   7  signed:   6  trust: 0-, 0q, 0n, 0m, 0f, 7u
+gpg: depth: 1  valid:   6  signed:   1  trust: 6-, 0q, 0n, 0m, 0f, 0u
+gpg: next trustdb check due at 2021-11-23
+pub   rsa4096 2021-04-07 [SC] [expires: 2023-04-07]
+      D9DEEABFF447B80F7EC03A4BBAA0A4837C891E29
+uid           [ultimate] Your Name <your.name@3mdeb.com>
+sig!3        BAA0A4837C891E29 2021-04-07  Your Name <your.name@3mdeb.com>
+sig!         B2EE71E967AA9E4C 2021-04-07  Piotr Król <piotr.krol@3mdeb.com>
+sub   rsa4096 2021-04-07 [E] [expires: 2023-04-07]
+sig!         BAA0A4837C891E29 2021-04-07  Your Name <your.name@3mdeb.com>
+
+gpg: 3 good signatures
+```
+
+Please send received key in encrypted email to your.name@3mdeb.com.
```

Comment:
For those who don't have a 3mdeb e-mail, is it okay to send this key to another e-mail?